connector/http-agent/app/Utils.js

```diff
@@ -1,8 +1,29 @@
+const createError = require('http-errors');
+
+const err = new createError.BadRequest();
+
 /**
  * A module with helper functions
  * @module utils
  */
 
+/**
+ * @function isInteger
+ *
+ * check if is integer
+ *
+ * @param {string} str
+ *
+ * @returns {boolean}
+ */
+
+const isInteger = (str) => {
+  if (typeof str !== 'string') return false; // we only process strings!
```

Comment:
If only process strings this is not an "isInteger" validation. Why not use "integer.parseint" ?